File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex18.java
//Importando Scanner
import java.util.Scanner;

public class Ex18 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("S = 1 + 1/2 + 1/4 + 1/6 + ... + 1/E\n\n");
		
		//vars
		double soma=1, i, e;
		
		//Entrada
		System.out.print("Digite o valor de E: ");
		e = input.nextDouble();
		
		//Processamento
		for ( i=2; i<=e; i=i+2 )
		{
			soma += 1/i;
		}
		
		//Saída
		System.out.println("Soma = " + soma);
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex18.c
#include <stdio.h>

main()
{
    printf("Soma 1 + 1/2 + 1/4 + 1/6 + ... + 1/E\n\n");

    //vars
    float e, i=2, soma=1;

    //Entrada
    printf("Digite o valor de E: ");
    scanf("%f",&e);

    //Processamento
    while(i<=e)
    {
        soma += 1/i;
        i= i+2;
    }

    //Saída
    printf("\nSoma = %.2f", soma);

}
